gnssanalysis/gn_utils.py

```diff
@@ -52,6 +52,31 @@ def get_filetype(path):
     return suffix
 
 
+def configure_logging(verbose: bool) -> None:
+    """Set up the logger object to use for encoding logging strings
+
+    :param bool verbose: Flag to increase level of detail to print to screen - True: DEBUG, False: INFO
+    """
+    if verbose:
```

Comment:
It's confusing (and conflicting) in `clkq` `verbose` being `_logging.INFO` and `else: _logging.disable()`, and so may be somewhere else. I would be better to unify them across the package.

Reply:
Just pushed a commit that uses the `configure_logging` function for all